apollo-entity/src/main/java/com/spotify/apollo/entity/EntityCodec.java

```diff
@@ -19,16 +19,15 @@
  */
 package com.spotify.apollo.entity;
 
-import com.spotify.apollo.Exploratory;
-
 import java.io.IOException;
-
 import okio.ByteString;
 
 /**
  * Interface for serializing and de-serializing entity types.
+ *
+ * @deprecated Use {@link Codec}
```

Comment:
Add "as that allows for content negotiation" or something similar?

Reply:
Done